model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/events/SessionEventsSenderTransaction.java

```diff
@@ -61,5 +61,5 @@ protected void rollbackImpl() {
         sessionEvents.clear();
     }
 
-    private record EventGroup(String eventKey, ClusterProvider.DCNotify dcNotify) {}
+    private record EventGroup(String eventKey) {}
```

Comment:
This class is redundant now. You can have `Map<String, List<ClusterEvent>> sessionEvents`